### DISCUSSION TITLE
Populating a schedule using workspace.yaml and multiple repositories

### DISCUSSION THREAD

**Discussion by blarghmatey** [dagster-io/dagster#2946] [Q&A] [answered]
I have got Dagit running using the Cron scheduler, and I am using a workspace.yaml for specifying the pipelines. I have two repositories set up that target the same pipeline, with separate schedules that are using different presets to allow for executing the pipeline against different source environments. When I attempt to execute `dagster schedule up -w /etc/dagster/workspace.yaml --location residential_edx_repository` it tells me that I need to configure a scheduler. The specific output is:
```
dagster schedule up --preview -w /etc/dagster/workspace.yaml --location residential_edx_repository
Usage: dagster schedule up [OPTIONS]

Error: A scheduler must be configured to run schedule commands.
You can configure a scheduler on your instance using dagster.yaml.
For more information, see:

https://docs.dagster.io/deploying/instance/#scheduler
```

**Comment by blarghmatey** [comment, marked as answer]
After some initial confusion where I assumed that I needed to pass the `--working-directory` flag and that telling me that I was creating an invalid command line, I realized that the missing ingredient was setting the `DAGSTER_HOME` environment variable.

**Reply by helloworld** [reply]
I'm glad you figured it out! We could do a better job with the error message here. I created this issue: https://github.com/dagster-io/dagster/issues/2949

**Comment by rexledesma** [comment]
Hey @blarghmatey - thanks for pointing this out, we updated the error message in https://github.com/dagster-io/dagster/commit/2c651b499fb1264117e04bfb384803d58c0b8717 to be more explicit to set `$DAGSTER_HOME`.
